library/Rules/NoWhitespace.php

```diff
@@ -5,6 +5,10 @@ class NoWhitespace extends AbstractRule
 {
     public function validate($input)
     {
+        if (is_array($input)) {
+            return false;
+        }
+
         return is_null($input) || !preg_match('#\s#', $input);
```

Comment:
> I guess it's better to use (is_scalar($input) && preg_match(.. instead of the checking above. What you think?

I think preserving behavior is better, I didn't quite understood you comment before. I am sorry.
Changing the rule not to accept `null` breaks BC which is never a good thing without a good reason.